apps/web-giddh/src/app/settings/permissions/form/form.component.html

```diff
@@ -1,143 +1,219 @@
+                [formGroupName]="i">
+                <div class="d-flex align-items-center">
+                    <div class="width-100">
+                        <input class="form-control " name="item_{{i}}" formControlName="range" type="text"
+                            placeholder="192.168.1.0/24" />
+                    </div>
+                    <div class="d-flex lh34 pl-05 pr-15">
+                        <i *ngIf="!l" (click)="delRow('allowedCidrs', i, $event)"
+                                    class=" icon-cross cursor-pointer"></i>
+
+                        <i *ngIf="l" (click)="addNewRow('allowedCidrs', item, $event)"
+                            class=" icon-add cursor-pointer"></i>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section formArrayName="allowedIps" class="col-xs-12"
+            *ngIf="permissionForm.get('ipOptions').value === 'ip_address'">
+            <label>Address</label>
+            <div class="d-flex align-items-center mb-1"
+                *ngFor="let item of permissionForm.get('allowedIps')['controls']; let i = index; let l = last"
+                [formGroupName]="i">
+                <div class="width-100">
+                    <input class="form-control" (keypress)="allowOnlyNumbersAndDot($event)" name="item_{{i}}"
+                        formControlName="range" type="text" placeholder="192.168.1.0" />
+                </div>
+                <div class="d-flex lh34 pl-05 pr-15">
+                    <i *ngIf="!l" (click)="delRow('allowedIps', i, $event)"
+                                class=" icon-cross cursor-pointer"></i>
+                    <i *ngIf="l" (click)="addNewRow('allowedIps', item, $event)"
+                        class="icon-add cursor-pointer"></i>
                 </div>
+            </div>
+        </section>
 
-                <section formArrayName="allowedCidrs" class="iprange container-fluid"
-                    *ngIf="permissionForm.get('ipOptions').value === 'cidr_range'">
-                    <div class="row mrB1"
-                        *ngFor="let item of permissionForm.get('allowedCidrs')['controls']; let i = index; let l = last"
-                        [formGroupName]="i">
-                        <div class="col-xs-11">
-                            <input class="form-control " name="item_{{i}}" formControlName="range" type="text"
-                                placeholder="192.168.1.0/24" />
-                        </div>
-                        <div class="pull-left lh34">
-                            <i *ngIf="!l" (click)="delRow('allowedCidrs', i, $event)"
-                                class="fa fa-trash-o cursor-pointer"></i>
-                            <i *ngIf="l" (click)="addNewRow('allowedCidrs', item, $event)"
-                                class="fa fa-plus cursor-pointer"></i>
+    <!--  -->
+    </div>
+    <div class="mb-4">
+    <p class="mb-15">
```

Comment:
Dummy text should be wrapped in ng-template
And new labels should be added to locale